nengo_spa/network.py

```diff
@@ -166,9 +166,6 @@ def get_network_input(self, name):
         except KeyError:
             raise SpaNetworkError("Could not find network input %r." % name)
 
-    def get_input_vocab(self, name):
```

Comment:
If you require the vocab of a module, it is better to use the `vocab`, `input_vocab`, or `output_vocab` parameters according to the module. These functions query the vocabularies associated with specific inputs or outputs in the `self.inputs` and `self.outputs` dictionaries which is more useful for internal use when connecting things up with action rules, but apparently not that useful anymore as these functions weren't used anymore.